app/experimenter/experiments/tests/test_models.py

```diff
@@ -845,15 +849,23 @@ def test_latest_returns_most_recent_changelog(self):
 
         self.assertEqual(experiment.changes.latest(), changelog2)
 
-    def test_str_uses_message_if_set(self):
-        change = ExperimentChangeLogFactory.create(message="Change Message")
-        self.assertEqual(str(change), "Change Message")
+    def test_pretty_status_created_draft(self):
+        experiment = ExperimentFactory.create_with_variants()
 
-    def test_str_uses_status_label_if_no_message(self):
-        change = ExperimentChangeLogFactory.create(
-            message=None, new_status=Experiment.STATUS_REVIEW
-        )
-        self.assertEqual(str(change), "Set Status: Ready for Sign-Off")
+        for old_status in ExperimentChangeLog.PRETTY_STATUS_LABELS.keys():
+            for new_status in ExperimentChangeLog.PRETTY_STATUS_LABELS[
+                old_status
+            ].keys():
+                expected_label = ExperimentChangeLog.PRETTY_STATUS_LABELS[
+                    old_status
+                ][new_status]
```

Comment:
Could simplify...

```python
for old_status, new_statuses in ExperimentChangeLog.PRETTY_STATUS_LABELS.items():
    for new_status, expected_label in new_statuses.items()
        changelog = ExperimentChangeLogFactory.create(
            experiment=experiment,
            old_status=old_status,
            new_status=new_status,
        )
        self.assertEqual(changelog.pretty_status, expected_label)
```

At least drop the redundant `.keys()`.

Reply:
Wow I don't know what I was on when I wrote that, you're right that's much more straight forward.